applications/search/src/components/search-concepts-hit-item/index.jsx

```diff
@@ -21,16 +21,12 @@ export default class ConceptsHitItem extends React.Component { // eslint-disable
   }
 
   _renderPublisher() {
-    const { publisher } = this.state.source;
-    if (publisher && publisher.name) {
+    const { creator } = this.state.source;
+    if (creator && creator.name) {
       return (
         <span className="inline-block">
           <strong>
-            {
-              (publisher && publisher.name)
-                ? publisher.name.charAt(0) + publisher.name.substring(1).toLowerCase()
-                : ''
-            }
+            { creator.name.charAt(0) + creator.name.substring(1).toLowerCase() }
```

Comment:
Er det slik at resten av tekst alltid skal være lowercase? 
Greit å sørge for alltid stor forbokstav, men er det ikke navn som skal ha kun store bokstaver (evt. NAV?) og navn som har flere ord (DItt og Datt)?
Eller er det sånn at et større problem er navn som ligger inne med bare store bokstaver (uten at de skal ha det)?

Reply:
Jeg tror du har rett. Det er vel ikke alltid at resten av navnet skal være lowercase, spesielt ved forkortelser. Jeg fjerner dette, i frontend må vi da forvente at det kommer på riktig form.